packages/cli/test/integration/users.api.test.ts

```diff
@@ -238,108 +243,309 @@ describe('GET /users', () => {
 
 describe('DELETE /users/:id', () => {
 	let owner: User;
-	let ownerPersonalProject: Project;
-	let member: User;
-	let memberPersonalProject: Project;
 	let ownerAgent: SuperAgentTest;
 
 	beforeAll(async () => {
 		await testDb.truncate(['User']);
 
 		owner = await createOwner();
-		ownerPersonalProject = await projectRepository.getPersonalProjectForUserOrFail(owner.id);
-
-		member = await createMember();
-		memberPersonalProject = await projectRepository.getPersonalProjectForUserOrFail(member.id);
 		ownerAgent = testServer.authAgentFor(owner);
 	});
 
 	test('should delete user and their resources', async () => {
-		const savedWorkflow = await createWorkflow({ name: randomName() }, member);
+		//
+		// ARRANGE
+		//
+		// @TODO: Include active workflow and check whether webhook has been removed
```

Comment:
I can give this a stab.

Reply:
Not sure if this would be the scope of this test, as it's part of the deletion process